rts/at/annotation/pom.xml

```diff
@@ -0,0 +1,82 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jboss.narayana.quickstart.rts</groupId>
+        <artifactId>rts-quickstarts</artifactId>
+        <version>6.0.2.Final-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <packaging>pom</packaging>
+    <artifactId>annotation</artifactId>
+    <name>REST-AT Annotation Example</name>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-bom</artifactId>
+                <version>${version.quarkus}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+   <modules>
+        <module>common-service</module>
+        <module>flight-service</module>
+        <module>hotel-service</module>
+        <module>trip-service</module>
+  </modules>
+
+    <properties>
+        <failOnMissingWebXml>false</failOnMissingWebXml>
+        <narayana.nodename.property>CoreEnvironmentBean.nodeIdentifier</narayana.nodename.property>
+        <surefire-plugin.version>2.22.0</surefire-plugin.version>
+        <version.quarkus>3.0.0.Alpha5</version.quarkus>
+        <jakarta.ws.rs.version>3.1.0</jakarta.ws.rs.version>
+    </properties>
+
+    <build>
+        <finalName>rest-at-annotation-service</finalName>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
I cannot see any surefire tests in these quickstarts so it is superfluous. Will you check all of the new poms for unused items, not just this plugin.

Reply:
Hi @mmusgrov , I've updated the changes, plz resolve the conversation it met the expectation. Thanks.